webpack.config.js

```diff
@@ -12,10 +13,21 @@ module.exports = {
     },
     module: {
         loaders: [
-            { test: /\.css$/,  loader: 'style!css' },
+            {
+                test:   /\.css$/,
+                loader: "style!css!postcss-loader"
```

Comment:
Consistency: always use single quotes, unless your string contains a single quote itself.

I recommend single over double quotes to be consistent with languages that use double quotes for interpolating strings: JS doesn't interpolate with either, so don't mislead.